.github/workflows/rust.yml

```diff
@@ -55,7 +55,22 @@ jobs:
         with:
           command: build
           args: --target wasm32-unknown-unknown --package mint-client
-
+  build_and_cli_test:
+    name: CLI Tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: Swatinem/rust-cache@v1
+      - uses: cachix/install-nix-action@v15
+        with:
+          nix_path: nixpkgs=channel:nixos-22.05
+      - uses: cachix/cachix-action@v10
+        if: github.repository_owner == 'fedimint'
```

Comment:
tl;dr we can get rid of the if

I just merged #214 because the docs and code of the cachix action indicate that it will already do the right thing if no auth token is present. The only problem was that the fork had a own token set and that wasn't valid for out repo so the test failed on the fork.